eng/pipelines/coreclr/perf-wasm-prepare-artifacts-steps.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  configForBuild: 'Release'
+
+steps:
+  - script: "./dotnet.sh build -p:TargetOS=Browser -p:TargetArchitecture=wasm -p:Configuration=${{ parameters.configForBuild }} /nr:false /p:TreatWarningsAsErrors=true /p:ContinuousIntegrationBuild=true src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj /t:InstallWorkloadUsingArtifacts"
```

Comment:
I don't think I understand what is actually happening here. Is this installing a workload in some dotnet host? Where did we get the workload; from the just built dotnet-runtime?

Reply:
Yeah, we install a new sdk with version from `eng/Versions.props` $(SdkVersionForWorkloadTesting)`. then install the workload manifests, and packs from the nugets generated during the runtime build, to this sdk.
Then use this sdk here. It is also used to run Wasm.Build.Tests .